.github/workflows/seeder.yml

```diff
@@ -22,18 +26,14 @@ jobs:
     steps:
       - name: Checkout Seeder
         uses: actions/checkout@v4
-        with:
-          path: ./seeder
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
           cache: "npm"
-        working-directory: ./seeder
+          cache-dependency-path: seeder/package-lock.json
```

Comment:
Very unnecessary

Reply:
https://github.com/coreproject-moe/CoreProject/actions/runs/10337853761/job/28615192230?pr=3962#step:3:15
https://dev.to/shofol/run-your-github-actions-jobs-from-a-specific-directory-1i9e#comment-1mo5m